#Notice: use only reasonable values, something like 99.99.99 is not possible,
#So expect some strange values on MFD if doing otherwise... 39.39.39 should work
uint8 date_field_left
uint8 date_field_center
uint8 date_field_right
